src/mokaQtIhm/object-options/options-frame.qt.cc

```diff
@@ -63,49 +63,49 @@ OptionsFrame :: OptionsFrame ( Window * parent )
 
-  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F6 ) , 6 ) ;
-  Raccourci -> connectItem ( 6 , this , SLOT ( callbackShowInterpolation()));
+//  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F6 ) , 6 ) ;
+//  Raccourci -> connectItem ( 6 , this , SLOT ( callbackShowInterpolation()));
 
-  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F7 ) , 7 ) ;
-  Raccourci->connectItem(7, this, SLOT(callbackShowDivers()));
+//  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F7 ) , 7 ) ;
+//  Raccourci->connectItem(7, this, SLOT(callbackShowDivers()));
   
-//  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F8 ) , 15 ) ;
-//  Raccourci -> connectItem ( 15 , this , SLOT ( callbackShowDivers ( ) ) ) ;
+////  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F8 ) , 15 ) ;
+////  Raccourci -> connectItem ( 15 , this , SLOT ( callbackShowDivers ( ) ) ) ;
 
-  Raccourci -> insertItem ( QKeySequence ( "Alt+F9" ) , 8 ) ;
-  Raccourci -> connectItem ( 8 , this , SLOT ( callbackToggleNormal ( ) ) ) ;
+//  Raccourci -> insertItem ( QKeySequence ( "Alt+F9" ) , 8 ) ;
+//  Raccourci -> connectItem ( 8 , this , SLOT ( callbackToggleNormal ( ) ) ) ;
 
-  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F9 ) , 9 ) ;
-  Raccourci -> connectItem ( 9 , this , SLOT ( callbackToggleSews ( ) ) ) ;
+//  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F9 ) , 9 ) ;
+//  Raccourci -> connectItem ( 9 , this , SLOT ( callbackToggleSews ( ) ) ) ;
 
-  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F10 ) , 10 ) ;
-  Raccourci -> connectItem ( 10 , this , SLOT ( callbackToggleVertices ( ) ) ) ;
+//  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F10 ) , 10 ) ;
+//  Raccourci -> connectItem ( 10 , this , SLOT ( callbackToggleVertices ( ) ) ) ;
 
-  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F11 ) , 11 ) ;
-  Raccourci -> connectItem ( 11 , this , SLOT ( callbackToggleFaces ( ) ) ) ;
+//  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F11 ) , 11 ) ;
+//  Raccourci -> connectItem ( 11 , this , SLOT ( callbackToggleFaces ( ) ) ) ;
 
-  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F12 ) , 12 ) ;
-  Raccourci -> connectItem ( 12 , this , SLOT ( callbackToggleGrille ( ) ) ) ;
+//  Raccourci -> insertItem ( QKeySequence ( Qt::Key_F12 ) , 12 ) ;
+//  Raccourci -> connectItem ( 12 , this , SLOT ( callbackToggleGrille ( ) ) ) ;
 
-  Raccourci->insertItem ( QKeySequence ( Qt :: Key_Space ) , 13 ) ;
-  Raccourci->connectItem(13, parent, SLOT (callbackHideAllWindow()));
+//  Raccourci->insertItem ( QKeySequence ( Qt :: Key_Space ) , 13 ) ;
+//  Raccourci->connectItem(13, parent, SLOT (callbackHideAllWindow()));
 
-  Raccourci -> insertItem ( QKeySequence ( "Alt+F11" ) , 14 ) ;
-  Raccourci -> connectItem ( 14 , this , SLOT (callbackTournerButton() ))  ;
+//  Raccourci -> insertItem ( QKeySequence ( "Alt+F11" ) , 14 ) ;
+//  Raccourci -> connectItem ( 14 , this , SLOT (callbackTournerButton() ))  ;
   
```

Comment:
I think some of those shortcuts (those regarding the tabs of the options window) are not really useful, since the main-window shortcuts are overriding them and provide the same functionality anyway. But I guess it doesn't harm to keep them.